newui/docs/_data/navigation.yml

```diff
@@ -0,0 +1,51 @@
+
+# main links
+main:
+  - title: "Documentation"
+    url: /docs/quick-start-guide.html
+  - title: "Community"
+    url: /community.html
+  - title: "Roadmap"
+    url: /roadmap.html
+  - title: "Activities"
+    url: /activity.html
+  - title: "FAQ"
+    url: https://cwiki.apache.org/confluence/display/HUDI/FAQ
+  - title: "Releases"
+    url: /releases.html
+
+# doc links
+docs:
+  - title: Getting Started
+    children:
+      - title: "Quick Start"
+        url: /docs/quick-start-guide.html
+      - title: "Structure"
```

Comment:
lets remove this page and incorporate its content onto eithr the current home page or skip

Reply:
Done